Kinvey.Core/Model/Entity.cs

```diff
@@ -32,7 +32,7 @@ public class Entity : IPersistable
         [DataMember(Name = "_id")]
         [Preserve]
 		[PrimaryKey, Column("_id")]
-		public string ID { get; set; }
+		public string Id { get; set; }
```

Comment:
@romanogolikhin Since this is a `public` property, do we have to deprecate this?

Reply:
Also, according to this [Microsoft doc](https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-1.1/141e06ef(v=vs.71)), for acronyms of only 2 letters, it should be all capitalized. Should we revert this?

Reply:
> @vinaygahlawat Sorry, but I misunderstood your idea. What did you mean?
> As far as I can see the task was about renaming only.

So because these properties are `public`, developers using this SDK may actually access these properties. Changing the name of this would break their code, so we should consider deprecating this if possible.

Reply:
> Also, according to this [Microsoft doc](https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-1.1/141e06ef(v=vs.71)), for acronyms of only 2 letters, it should be all capitalized. Should we revert this?

@vinaygahlawat  I have read the article. I agree with it. We need to revert changes for "Id". It should be 'ID'.




> > @vinaygahlawat Sorry, but I misunderstood your idea. What did you mean?
> > As far as I can see the task was about renaming only.
> 
> So because these properties are `public`, developers using this SDK may actually access these properties. Changing the name of this would potentially break their code, so we should consider deprecating this if possible.

You mean changing of property name or its value?

Reply:
It seems to me. I understood you.

Reply:
@vinaygahlawat You mean if we release this changes, our clients will have problem in their custom code where they use them. Is it correct?

Reply:
@vinaygahlawat 
I had a look at it deeply. The situation is complicated as the 'Entity' inherits a "IPersistable" interface and some attributes([JsonProperty("_acl")], [DataMember(Name = "_acl")],[Preserve],[Column("_acl")])).
It means if we put [obsolete] attribute to ACL and KMD, we will need to extend  the "IPersistable" interface as some customers were able to use these properties through the interface. Extending of the interface it is bad way for us.
Maybe you had such experience early in Kinvey SDK and can give some advice?  

I saw we used [obsolete] attribute in some method and properties. But they were easy cases.